packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -43,7 +43,14 @@ class _LeastSquaresVelocityTrackerStrategy extends _VelocityTrackerStrategy {
   int _index = 0;
 
   static const int kHistorySize = 20;
-  static const int kHorizonMilliseconds = 40;
+
+  // On iOS we're not necccessarily seeing all of the motion events. See:
+  // https://github.com/flutter/flutter/issues/4737#issuecomment-241076994
```

Comment:
This comment may belong somewhere higher with a TODO, rather than next to this seemingly unrelated constant.